suggestions/tts.json

```diff
@@ -4,5 +4,11 @@
         "label": "TTS Util",
         "source": "fg",
         "repo": "https://github.com/Danesprite/tts-util-app"
+    },
+    {
+        "id": "com.github.olga_yakovleva.rhvoice.android",
+        "label": "RHVoice",
+        "source": "f",
```

Comment:
Also available on  [Play Store](https://play.google.com/store/apps/details?id=com.github.olga_yakovleva.rhvoice.android).
Add value `g`

Reply:
It's fake on play store

Reply:
Check versions

Reply:
+ the actual site is RHVoice.org but in play store it says RHVoice.com

Reply:
Doesn't matter, that's the official one source: https://rhvoice.org/installation/#android

Reply:
I think it's outdated

Reply:
Updated on : Jul 16, 2023